tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "lib": [
+      "es2019",
+      "DOM"
```

Comment:
nit: this is running on Node, so I don't think we want to specify that DOM libraries are available.

you also may want to `yarn add -D @types/node` since those typings aren't available by default.